docs/content/concepts/scheduler.md

```diff
@@ -127,7 +134,38 @@ the flow of incoming requests. In this algorithm, every flow is required to
 obtain tokens from the bucket before a specified deadline to gain admission to
 the system.
 
-### Timeout Factor {#timeout-factor}
+The Scheduler employs two distinct types of token buckets: the Dynamic Token
+Bucket and the Fixed Token Bucket. These buckets, each with a unique method of
+operation, enhance the system's capacity to effectively adapt to various
+scheduling scenarios.
+
+#### Dynamic Token Bucket {#dynamic-bucket}
+
+The Dynamic Token Bucket is primarily used for service protection. It adjusts
+the token rate in response to the changing load signal, thereby ensuring an
+active defense for the service. Each agent maintains an individual token bucket,
+and synchronization between agents happens in lazy sync. This dynamic resizing
+enables efficient management of requests and maintains optimal service
+protection levels.
+
+#### Fixed Token Bucket {#fixed-bucket}
+
+In contrast to the Dynamic Token Bucket, the Fixed Token Bucket operates with a
+predetermined number of tokens, establishing a constant token rate. This type of
+bucket is particularly advantageous when the load or the number of tokens is
+known upfront.
+
+The Fixed Token Bucket can be employed for service protection but is especially
+useful in scenarios where there are strict rate limits. For instance, when
+scheduling access to external APIs with a global quota, the Fixed Token Bucket
+ensures that the API's rate limits are adhered to.
+
+Both the Dynamic and Fixed Token Buckets serve to manage and control the flow of
+requests, their specific usage and operation depend on the requirements and
+constraints of the system, providing versatility and adaptability in handling
+workloads.
+
+### Queue timeout {#queue-factor}
 
 The timeout factor parameter determines the duration a request in the workload
```

Comment:
Timeout factor is deprecated. The check call’s timeout (or per workload timeout in the future) is used instead